recipe/build.sh

```diff
@@ -5,8 +5,9 @@ set -ex
 # remove outdated vendored headers
 rm -rf $SRC_DIR/python/triton/third_party
 
+export TRITON_OFFLINE_BUILD=1
```

Comment:
could you leave a comment what this is supposed to do?

Reply:
Done.